_layouts/default.html

```diff
@@ -6,18 +6,11 @@
     <body>
       <div id="wrapper">
 
-		      <div id="header">
```

Comment:
Just letting you know had the ID wrappers out here to keep each page consistent, plus it solved the problem you're have having with the footer right now. It basically stack each element on top of eachother.

If you want to keep the same, here is an article for other methods of keeping the footer at the bottom of the page:
https://css-tricks.com/couple-takes-sticky-footer/